tsl/src/bgw_policy/compression_api.c

```diff
@@ -149,7 +149,7 @@ policy_compression_add(PG_FUNCTION_ARGS)
 		ts_cache_release(hcache);
 		ereport(ERROR,
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-				 errmsg("can add compress_chunks policy only on hypertables with compression "
+				 errmsg("can add compression policy only on hypertables with compression "
```

Comment:
I find the formulation here can be improved. Realize it was there before, but maybe we can take the opportunity to improve this:


```
errmsg("compression not enabled on hypertable \"%s\"", ....),
errhint("Enable compression before adding a compression policy.")
```

Reply:
Improved